### ISSUE TITLE
This fails if the url is external

### ISSUE THREAD

**Issue by pelusanchez** [ecmwf-projects/cads-catalogue#111]
For example, if we have this image:

```
https://github.com/ecmwf-projects/cads-forms-cim-json/blob/test/projections-cmip6/quality_assurance.layout.json#L15
```

This will fail:
```
https://github.com/ecmwf-projects/cads-catalogue/blob/749027a9c65c8c0e00ad1a6693802bdc8a70a134/cads_catalogue/layout_manager.py#L82C1-L82C17
```

Can we download the image?

**Comment by EddyCMWF** [comment]
Hi, we also have issues downloading images from the CIM object store, here is a traceback from our dev instance:

```
Traceback (most recent call last):
  File "/src/cads-catalogue/cads_catalogue/manager.py", line 633, in update_catalogue_resources
    resource = layout_manager.transform_layout(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/src/cads-catalogue/cads_catalogue/layout_manager.py", line 410, in transform_layout
    layout_data = transform_image_blocks(
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/src/cads-catalogue/cads_catalogue/layout_manager.py", line 123, in transform_image_blocks
    sections[i] = manage_image_section(
                  ^^^^^^^^^^^^^^^^^^^^^
  File "/src/cads-catalogue/cads_catalogue/layout_manager.py", line 84, in manage_image_section
    blocks[i] = manage_image_section(
                ^^^^^^^^^^^^^^^^^^^^^
  File "/src/cads-catalogue/cads_catalogue/layout_manager.py", line 82, in manage_image_section
    raise ValueError(f"image {image_rel_path} not found")
ValueError: image https://object-store.os-api.cci2.ecmwf.int/bopen-backoffice-images/cim-dev/document_82844df8-6173-4213-a661-7ef57a95eda2 not found
```
What is odd is that the image does exist, just the catalogue-manager pod cannot see it

**Comment by alex75** [comment]
This should be closed with https://github.com/ecmwf-projects/cads-catalogue/pull/112

**Comment by pelusanchez** [comment]
> This should be closed with #112

Thank you Alessio!
